src/screens/Following.tsx

```diff
@@ -30,16 +31,27 @@ export default class LogIn extends Component<IFollowingProps, IFollowingState> {
   render() {
     return (
       <View style={FollowingStyles(this.context).screen.style.container}>
-        <Text style={FollowingStyles(this.context).screen.style.text}>
-          Open up ./src/screens/Following.tsx to start working on your app!
-        </Text>
-        <Button
-          title={'Go to Details'}
-          onPress={() => {
-            this.props.navigation.navigate('Details');
-          }}
+        <FlatList
+          style={FollowingStyles(this.context).screen.style.content}
+          data={new Array(10)}
+          renderItem={() => (
+            <TouchableOpacity
+              style={{ marginVertical: 5, marginHorizontal: 20 }}
+              onPress={() => {
+                this.props.navigation.navigate('Details');
+              }}
+            >
+              <CWCard
+                style={{ ...FollowingStyles(this.context).weatherCard }}
+                data={{
+                  city: 'Colima',
+                  degrees: 80,
+                  icon: 'https://www.losmundosdenoa.es/wp-content/uploads/2020/06/sol-e1603880900911.png',
```

Comment:
I understand, the thing is that this information is only provisional, because it will be loaded from the database. So, should we add the file even if we are going to remove it when the database is ready?

Reply:
I got it, in that case I don't have any problem